api-reference/50 Common/Object Structures/positionConfig/boundary.md

```diff
@@ -4,7 +4,9 @@ type: String | Node | jQuery | Window
 ---
 ---
 ##### shortDescription
-The element within which the widget is positioned.
+A boundary element within which the overlay element should be positioned.
 
 ---
-All collisions are computed against the element passed to this option. By default, widgets use the window object as a boundary element.
\ No newline at end of file
+The boundary element limits possible positions for the overlay element. The default boundary element is <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window">Window</a>. If the actual boundaries should be narrower or wider than the boundary element, set the [boundaryOffset](/Documentation/ApiReference/Common/Object_Structures/positionConfig/boundaryOffset/). Whenever the specified position violates the boundaries, a [collision](/Documentation/ApiReference/Common/Object_Structures/positionConfig/collision/) occurs.
```

Comment:
I'm not  really sure, but "exceeds the boundaries" might be a better choice than "violates the boundaries", that is mostly used in psychology.